workitem.go

```diff
@@ -114,6 +114,42 @@ func (c *WorkitemController) Update(ctx *app.UpdateWorkitemContext) error {
 	})
 }
 
+// Reorder reorders the workitem
+func (c *WorkitemController) Reorder(ctx *app.ReorderWorkitemContext) error {
+	return application.Transactional(c.db, func(appl application.Application) error {
+		var dataArray []*app.WorkItem2
+		if ctx.Payload == nil || ctx.Payload.Data == nil {
+			jerrors, _ := jsonapi.ErrorToJSONAPIErrors(errors.NewBadParameterError("data", nil))
+			return ctx.NotFound(jerrors)
```

Comment:
This looks like an error. Why do you construct a bad parameter error and then send a not found error down the line?